packages/metascraper-description/README.md

```diff
@@ -11,6 +11,19 @@
 $ npm install metascraper-description --save
 ```
 
+## API
+
+### description([options])
+
+#### options
+
+##### truncateLength
+
+Type: `string`<br>
+Default: `300`
```

Comment:
should this be a string type or number?

Reply:
If this is the way to add the option, then wouldn't it be a number?
const metascraper = require('metascraper')([
  require('metascraper-description')({
    truncateLength: 500
  })
])

Reply:
yes!